docs/python/guides/flags.md

```diff
@@ -5,4 +5,238 @@ sidenav: side-nav-python.html
+    `FLAGS.myflag` to `False`. `--myflag=true` and `--myflag=false` are also
+    supported, but not recommended.
+*   `DEFINE_float`: takes an input and interprets it as a floating point number.
+    Takes optional args lower_bound and upper_bound; if the number specified on
+    the command line is out of range, it will raise a `FlagError`.
+*   `DEFINE_integer`: takes an input and interprets it as an integer. Takes
+    optional args lower_bound and upper_bound as for floats.
+*   `DEFINE_enum`: takes a list of strings which represents legal values. If the
+    command-line value is not in this list, raise a flag error. Otherwise,
+    assign to `FLAGS.flag` as a string.
+*   `DEFINE_list`: Takes a comma-separated list of strings on the commandline.
+    Stores them in a Python list object.
+*   `DEFINE_spaceseplist`: Takes a space-separated list of strings on the
+    commandline. Stores them in a Python list object. Example: `--myspacesepflag
+    "foo bar baz"`
+*   `DEFINE_multi_string`: The same as DEFINE_string, except the flag can be
+    specified more than once on the commandline. The result is a Python list
+    object (list of strings), even if the flag is only on the command line once.
+*   `DEFINE_multi_integer`: The same as DEFINE_integer, except the flag can be
+    specified more than once on the commandline. The result is a Python list
+    object (list of ints), even if the flag is only on the command line once.
+*   `DEFINE_multi_enum`: The same as DEFINE_enum, except the flag can be
+    specified more than once on the commandline. The result is a Python list
+    object (list of strings), even if the flag is only on the command line once.
+
+## Special Flags
+
+There are a few flags that have special meaning:
+
+*   `--help`: prints a list of all key flags (see below).
+*   `--helpshort`: alias for `--help`.
+*   `--helpfull`: prints a list of all the flags in a human-readable fashion.
+*   `--helpxml`: prints a list of all flags, in XML format. DO NOT parse the
+    output of `--helpfull` and `--helpshort`. Instead, parse the output of
+    `--helpxml`. For more info, see "OUTPUT FOR --helpxml" below.
+*   `--flagfile=foo`: read flags from file foo.
+*   `--undefok=f1,f2`: ignore unrecognized option errors for f1,f2. For boolean
+    flags, you should use `--undefok=boolflag`, and `--boolflag` and
+    `--noboolflag` will be accepted. Do not use `--undefok=noboolflag`.
+*   `--`: as in getopt(), terminates flag-processing.
+
+## Implementation
+
+`DEFINE_*` creates a `Flag` object and registers it with a `FlagValues` object
+(typically the global FlagValues FLAGS, defined in `__init__.py`). The
+`FlagValues` object can scan the command line arguments and pass flag arguments
+to the corresponding `Flag` objects for value-checking and type conversion. The
+converted flag values are available as attributes of the `FlagValues` object.
+
+Code can access the flag through a `FlagValues` object, for instance
```

Comment:
the flag through a `FlagValues` object, for instance -> a flag through a `FlagValues` object.

I'd add a code sample below then, showing flags.FLAGS.myflag and a comment.